README.md

```diff
@@ -1 +1,169 @@
-# Capstone
\ No newline at end of file
+## Capstone Project
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run Linters](#run-linters)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone Project <a name="about-project"></a>
+
+> Capstone Project is designed to provide information and resources to students, faculty members, and university visitors, showcasing the academic programs and research opportunities available. Built with HTML, CSS, and JavaScript. The main objective of this project is to start practicing the professional way to work with CSS HTML and JavaScript in responsive design, using the basic structure, data query, flexbox and all following a template in Behance. Capstone Project
+
+![image](./resources/image/Highlights.png)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li><a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add the linter for the project to fix the errors and clean the codeok_hand
+- User-friendly Interface: Simple and intuitive interface for a seamless user experience.rocket
+- Style the text using the CSS 100
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://www.appu-bureau.org/wp-content/uploads/2021/09/coming-soon-animated-gif-8.gif)
```

Comment:
- [ Optional ] I think it would be better if you deploy your application online using GitHub pages and update the link to your project on the readme file. Deploying the application online will make it easy for recruiters who have no skills in setting up applications to see your work online.

Reply:
Hi @Uzair-Manzoor ! Even if it is mentioned as optional by the previous review. I recommend you deploy this application and add the link here in readme file. It was one of the project requirements and will make your project accessible to the recruiters. Kindly add the link before merging this Pull request.